decidim-core/app/assets/stylesheets/decidim/modules/_author-avatar.scss

```diff
@@ -7,168 +7,169 @@ $author-data-color: $muted;
 
 //Styles
 
-.author-data{
+.author-data {
```

Comment:
Why are spaces added between the selector and the curly brace? Isn't this something CodeClimate complaints about?